test/retention-cleanup.test.ts

```
@@ -0,0 +1,93 @@
+
+		const root = mkdtempSync(path.join(tmpdir(), "retention-fail-"));
+		fixtures.push(root);
+		const logsDir = path.join(root, "logs");
+		await fs.mkdir(logsDir, { recursive: true });
+		const stalePath = path.join(logsDir, "stale.log");
+		await fs.writeFile(stalePath, "stale\n", "utf8");
+		const staleDate = new Date(Date.now() - 10 * 24 * 60 * 60 * 1000);
+		utimesSync(stalePath, staleDate, staleDate);
+
+		chmodSync(logsDir, 0o500);
+		let result;
```

Biome (2.4.4)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)